lib/libzfs/os/linux/libzfs_pool_os.c

```diff
@@ -236,7 +236,7 @@ zpool_label_disk(libzfs_handle_t *hdl, zpool_handle_t *zhp, const char *name)
 		start_block = NEW_START_BLOCK;
 	}
 
-	(void) snprintf(path, sizeof (path), "%s/%s", DISK_ROOT, name);
+	(void) snprintf(path, sizeof (path), "%s%s", DISK_ROOT, name);
```

Comment:
What is the reasoning here?

Reply:
```
#ifdef _WIN32
#define	DISK_ROOT	"\\\\?\\"
#else
#define	DISK_ROOT	"/dev/"
#endif
```
Basically, the oooooold "/" vs "\\" again. So I stuck the "/" in the #define instead and remove it from the code. Windows can not handle `"\\\\?/"` nor  `"\\\\?\\/"`.  Traditionally, other projects have gone with PATH_SEPARATOR style macro, but this was the only slash that got in the way.